packages/core/story-structure.js

```diff
@@ -140,6 +140,10 @@ const s = [
           'c/TearsheetShell',
         ],
       },
+      {
+        n: 'Novice to pro',
+        s: ['c/NonLinearReading'],
```

Comment:
In the Storybook directory layout, **Novice to pro** has been designated as a folder directly beneath **IBM Products**

<img src="https://user-images.githubusercontent.com/52505287/227584770-29a68418-6e39-4a96-9d74-dce5e69319b8.png" width="225"/>